website/docs/advanced-concepts/multiple-environments.md

```diff
@@ -0,0 +1,95 @@
+
+# Multiple Environments
+
+Environments are container entities within a workspace that let you create different updates to different versions of your content types while still keeping them all in isolation. This process is often utilized when developing, identifying errors, or introducing new features without any concerns about how it might influence the current production.
+
+:::info
+Multiple Environments are only available on [Business Edition](https://www.appsmith.com/pricing). 
+:::
+
+Software developers can run numerous tests in these environments to filter out bugs and raise the application's overall quality. Different testing settings can be used to examine different aspects of a software application, including its performance, user interface, security, and more.
+
+## Environments on Appsmith
+There are two different environments that are available on Appsmith:
+
+### Staging environment
+Before launching an application, staging environments are created to evaluate builds, codes, and updates in an environment which is similar to a production environment. This is the area that developers take advantage of to inspect new aspects, debug, and explore their applications.
+
+:::info
+It's also referred to as the Testing or QA environment
+:::
+* The configuration values for a datasource are usually the same for staging and production. If you want to change them, you can go to the ```manage environments``` page and make the necessary adjustments.
```

Comment:
I feel we should omit this entire bullet section

Reply:
removed